cfgen/cfgen

```diff
@@ -264,10 +266,13 @@ def setup_logging(opts: Any) -> None:
     verbosity = logging.INFO
     if opts.verbose:
         verbosity = logging.DEBUG
-
-    logging.basicConfig(level=verbosity,
-                        handlers=handlers,
-                        format='%(asctime)s [%(levelname)s] %(message)s')
+    log_format = "%(asctime)s %(name)s %(process)d [%(levelname)s] " \
+                 "%(message)s"
+    formatter = logging.Formatter(fmt=log_format)
+    LOG.setLevel(verbosity)
+    for hndlr in handlers:
```

Comment:
I think `handler` or `h` is more readable and less prone to mistakes.

Reply:
changed name from hndlr to handler.

Reply:
Fixed.